Lib/test/test_sysconfig.py

```diff
@@ -387,10 +387,13 @@ def test_triplet_in_ext_suffix(self):
         if re.match('(i[3-6]86|x86_64)$', machine):
             if ctypes.sizeof(ctypes.c_char_p()) == 4:
                 self.assertTrue(suffix.endswith('i386-linux-gnu.so') or
-                                suffix.endswith('x86_64-linux-gnux32.so'),
+                                suffix.endswith('x86_64-linux-gnux32.so') or
+                                suffix.endswith('i386-linux-musl.so'),
                                 suffix)
```

Comment:
```suggestion
                                f'unexpected suffix {suffix!r}')
```

Reply:
Due to the deleted line I couldn't make a complete suggestion on using a tuple for `str.endswitch()` like I did below.